tools/devtool

```diff
@@ -745,7 +745,7 @@ cmd_test() {
       # anyway (something else on the host seems to be stealing our huge pages, and we cannot "ear mark" them for
       # Firecracker processes). Thus, just allocate 4GB of them and call it a day.
       say "Setting up huge pages pool"
-      num_hugetlbfs_pages=2048
+      num_hugetlbfs_pages=4096
```

Comment:
we should probably update the comment above about the size

Reply:
true, done!